donkeycar/parts/tub_v2.py

```diff
@@ -130,3 +132,38 @@ def shutdown(self):
         self.close()
 
 
+
+
+class TubWiper:
```

Comment:
This is part of the RCReceiver branch, pls see comment above.

Reply:
As mentioned, I need that in. It will soon be standard so no harm in having it

Reply:
Ok fine, just wanted your confirmation.